x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/task/agent/sub_graphs/translate_rule/nodes/translation_result/translation_result.ts

```diff
@@ -18,8 +19,9 @@ export const getTranslationResultNode = (): GraphNode => {
     const elasticRule = {
       title: state.original_rule.title,
       description: state.original_rule.description || state.original_rule.title,
-      severity: DEFAULT_TRANSLATION_SEVERITY,
-      risk_score: DEFAULT_TRANSLATION_RISK_SCORE,
+      risk_score:
+        getElasticRiskScoreFromOriginalRule(state.original_rule) || DEFAULT_TRANSLATION_RISK_SCORE,
+      severity: getElasticSeverityFromOriginalRule(state.original_rule),
```

Comment:
This is where we set the defaults in case they come undefined. Could we set `DEFAULT_TRANSLATION_SEVERITY` and `DEFAULT_TRANSLATION_RISK_SCORE` here, and use the `getElasticRiskScoreFromOriginalRule` and `getElasticSeverityFromOriginalRule` functions in the `translate_rule` node, where we create the custom translation?

Reply:
Thanks for the input. It makes sense. Moved to `translate_rule` node here: https://github.com/elastic/kibana/pull/211202/commits/a50a7ba47cbb9dc01beddb89743dc60d3ae204b2